content/en/platform/corda/4.8/open-source/key-concepts-node.md

```diff
@@ -70,33 +69,32 @@ are documented in [API: RPC operations](api-rpc.md).
 Internally, the node has access to a rich set of services that are used during flow execution to coordinate ledger
 updates. The key services provided are:
 
-* Information on other nodes on the network and the services they offer
-* Access to the contents of the vault and the storage service
-* Access to, and generation of, the node’s public-private keypairs
-* Information about the node itself
-* The current time, as tracked by the node
+* Information on other nodes on the network and the services they offer.
+* Access to the contents of the vault and the storage service.
+* Access to, and generation of, the node’s public-private key pairs.
+* Information about the node itself.
+* The current time, as tracked by the node.
 
 ## The CorDapp provider
 
 The CorDapp provider is where new CorDapps are installed to extend the behavior of the node.
 
-The node also has several CorDapps installed by default to handle common tasks such as:
+Several CorDapps are installed on the node by default to handle common tasks such as:
 
-* Retrieving transactions and attachments from counterparties
-* Upgrading contracts
-* Broadcasting agreed ledger updates for recording by counterparties
+* Retrieving transactions and attachments from counterparties.
+* Upgrading contracts.
+* Broadcasting agreed ledger updates for recording by counterparties.
 
 ## Draining mode
```

Comment:
This section needs to be refocused on core concepts. In this case, we need some more info. The header for this section should probably be something like "shutting down a node". We probably also need some info on spinning up a node. 

1) _Why_ drain a node? (to shut it down) Use cases?
2) What's a checkpoint/what does "in-flight" mean?
3) Who's a node owner? Who's a node operator? What's the difference, and what can each do?
4) What's the shell?